dashboard/client/src/pages/serve/ServeApplicationDetailPage.tsx

```diff
@@ -57,33 +61,19 @@ export const ServeApplicationDetailPage = () => {
     setPage,
     changeFilter,
     allDeployments,
-  } = useServeApplicationDetails(name);
+  } = useServeApplicationDetails(applicationName);
 
   if (!application) {
     return (
       <Typography color="error">
-        Application with name "{name}" not found.
+        Application with name "{applicationName}" not found.
       </Typography>
     );
   }
 
   const appName = application.name ? application.name : "-";
   return (
     <div>
-      {/* Extra MainNavPageInfo to add an extra layer of nesting in breadcrumbs */}
-      <MainNavPageInfo
-        pageInfo={{
-          id: "serveApplicationsList",
-          title: "Applications",
```

Comment:
no, this has to be here because Serve Application Page is a parent of ServeReplicaDetails and we need to have this nav bar show up for the detail page.